code/tools/make_munki_mpkg.sh

```diff
@@ -341,10 +346,11 @@ done
 # Set permissions.
 chmod -R go-w "$ADMINROOT/usr/local/munki"
 chmod +x "$ADMINROOT/usr/local/munki"
+
 # make paths.d file
 mkdir -p "$ADMINROOT/private/etc/paths.d"
 echo "/usr/local/munki" > "$ADMINROOT/private/etc/paths.d/munki"
-chmod -R 755 "$ADMINROOT/private"
+chmod -R 755 "$ADMINROOT/private/etc/paths.d"
```

Comment:
We're setting the mode on the files/directories we've created in our $ADMINROOT. Now the mode of private and etc are unclear. What are they?